src/App.tsx

```diff
@@ -1,14 +1,72 @@
 /* eslint-disable max-len */
-import React from 'react';
+import { useEffect, useMemo, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const [isLoading, setIsLoading] = useState(true);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+  const [todoSelector, setTodoSelector] = useState('all');
+  const [todoFilter, setTodoFilter] = useState('');
+
+  useEffect(() => {
+    getTodos().then((data) => {
+      setTodos(data);
+      setIsLoading(false);
+    });
```

Comment:
nb: would be better to use `async/await`

Reply:
> hey! GJ

Hey! Thanks, done!